README.md

```diff
@@ -51,24 +51,116 @@ If BSG decide to create the ability to do this on live OR I receive a DCMA reque
+
+### Client Install
+- **Everyone** must install the SIT Client Mod. You can install it using SIT Manager, or manually if desired.
+- **IF YOU USE SPT ALREADY**: Do **NOT** install the SIT Client mod onto your existing SPT install. The SIT Client Mod is currently not compatible with the SPT-Aki client, so it needs to be installed on it's own copy of Tarkov.
+
+#### SIT Manager Method
+- Copy the contents of your live EFT installation into the currently-empty `SIT/game` folder
+  - If you installed tarkov to the default location, it will be under `C:\Battlestate Games\EFT`.
+- Launch `SIT/launcher/SIT.Manager.exe`
+- Point the manager to the target game directory
+  - Open `Settings` in the bottom left
+  - Set `Install Path` to `X:\<Full_Path_To>\SIT\game`, or use the Change button and select the `SIT/game` folder 
+    - Replace `X` and `<Full_Path_To>` with the path to the folder.
+  - Close Settings
+- Open the `Tools` menu on the left, and select `+ Install SIT`
+- Select the desired SIT version (choose the latest if you don't know what you're doing)
+- Click `Install`  
+
+<details>
+ <summary>Manual install method</summary>
+Note that these are the same steps the SIT Manager performs. If you don't have any reason to, you should probably just use the SIT manager- it's so much quicker and easier. (Seriously, we're not hiding anything from you here. These steps are literally just a plain-english description of [the manager code](https://github.com/stayintarkov/SIT.Manager/blob/master/SIT.Manager/Classes/Utils.cs#L613))
+
+- Copy the contents of your live EFT installation into the currently-empty `SIT/game` folder
+  - If you installed tarkov to the default location, it will be under `C:\Battlestate Games\EFT`.
+- Create the folowing directories in `SIT/game`:
+  - `SITLauncher/`
+  - `SITLauncher/CoreFiles/`
+  - `SITLauncher/Backup/Corefiles/`
+- Download the desired version of the client-mod from this repo's [releases page](https://github.com/stayintarkov/StayInTarkov.Client/releases) to `SIT/game/SITLauncher/CoreFiles` (choose the latest if you don't know what you're doing)
+  - Create the folder `SIT/game/SITLauncher/CoreFiles/StayInTarkov-Release/`
+  - Extract the contents of the release archive into that folder
+- Clean up your `SIT/game` directory
+  - Delete the following files & directories:
+    - `BattleEye/` \*
+    - `EscapeFromTarkov_BE.exe` \*
+    - `cache/`
+    - `ConsistencyInfo`
+    - `Uninstall.exe`
+    - `Logs/`
+  - \* In case of concern, note that this is not a method that can be used to cheat in live tarkov. SPT (and SIT, by extension) don't use the BattleEye executables/files because the SPT-Aki server does not run battleeye.
+    Please be careful, and don't delete these files from your live directory. At best you'll brick your install & won't be able to connect to live servers. At worst you'll trigger a BattleEye detection and get your Account/IP/HWID marked for doing.
```

Comment:
Tone revisions per @trippyone's request.

I reread thru a couple of times, and I'm pretty sure this was the only instance of language that could be considered objectively 'unprofessional' here.